yb-voyager/src/srcdb/postgres.go

```diff
@@ -121,7 +124,7 @@ func (pg *PostgreSQL) getConnectionUri() string {
 }
 
 func (pg *PostgreSQL) ExportSchema(exportDir string) {
-	pgdumpExtractSchema(pg.getConnectionUri(), exportDir)
+	pgdumpExtractSchema(pg.source, pg.getConnectionUri(), exportDir)
```

Comment:
Why not pass list of schema names as argument instead of sending in the entire `source`?

Reply:
hmm, we can do that. Actually i was thinking if something new comes tomorrow we don't have to make function signature complete by passing more and more variables.